index.js

```diff
@@ -111,18 +111,18 @@ export class Placement {
    * string from API response. Can also be null, indicating a noop action.
    */
   fetch() {
-    const id = "ad_" + Date.now();
+    const callback = "ad_" + Date.now();
     const url_params = new URLSearchParams({
       publisher: this.publisher,
       ad_types: this.ad_type,
-      div_ids: id,
-      callback: id,
+      div_ids: this.target.id,
```

Comment:
In most of our setups including on ethicalads.io, this div does not have an ID. I don't think this will work.

Reply:
Yea, that's fine. It's only needed if we have multiple placements. I had local changes I didn't push up that address this, but just sets a specific `undefined` id for these cases.